Cargo.toml

```diff
@@ -123,6 +124,7 @@ edition = "2021"
 rust-version = "1.66.0"
 
 [workspace.dependencies]
+wasm-valgrind = { path = "crates/valgrind", version = "0.1.0" }
```

Comment:
Let's call this `wasmtime-valgrind` for consistency with the other crates in `crates/`; also, the version should match the others (`11.0.0` here but likely `13` once you merge or rebase to a latest `main`).